contrib/xfce4/template.py

```diff
@@ -0,0 +1,50 @@
+pkgname = "xfce4"
+pkgver = "4.18"
+pkgrel = 0
+build_style = "meta"
+depends = [
+    "adw-gtk3",
+    "adw-xfwm4",
```

Comment:
the theme dependencies should be on the things that declare that the theme is used

Reply:
sorry i don't quite get what that means, can you rephrase?

Reply:
the patch for using adw-gtk3 is in xfce4-settings so it should be depended on in that instead of in xfce4

same for the others that do something similar

Reply:
makes sense yeah

Reply:
though i don't think there's anything in particular that depends on `fonts-dejavu` so i'm not sure where to put that, or perhaps we can have a `base-desktop-xfce4` which would pull in `base-desktop` which does bring along some font packages?